### PR TITLE
YLP-111 Display localized CGU and data privacy

### PR DESCRIPTION
**Pull request by yvon-dblg**
May be better to merge https://github.com/mdblp/blip/pull/231 & https://github.com/mdblp/blip/pull/223 before.
I will have some conflicts to resolves, and some changes to do too.

Not really sure of the formats.

### CODE REVIEW

File: app/bootstrap.js [hunk 1]
```diff
@@ -9,95 +9,138 @@
...
+        const countryCode = _.get(this.terms, `defaultCountries.${lang}`, null);
+        legals = _.get(this.terms, `${countryCode}.${lang}`, null);
+      }
+      if (legals === null) {
+        this.log.warn(`not terms for defaultCountries.${lang}`);
+        legals = _.get(this.terms, 'defaults', null);
+      }
+      if (legals !== null) {
+        _.assign(BrandConfig.diabeloop, legals);
+        this.log.info('brandin', BrandConfig.diabeloop);
```
Comment (yvon-dblg) [inline app/bootstrap.js:93]:
```suggestion
        this.log.info('branding', BrandConfig.diabeloop);
```

File: server/server.js [hunk 2]
```diff
@@ -55,23 +72,32 @@ function redirectMiddleware(req, res, next) {
...
+        // Do a redirect to the file -> can be in cache in the browser.
+        return res.redirect(`/${filePath}`);
+      }
+      filePath = `${pathParts.pop()}/${filePath}`;
     }
-    // Do a redirect to the file -> can be in cache in the browser.
-    return res.redirect(`/${file}`);
   }
 
+  console.log('default index.html');
```
Comment (yvon-dblg) [inline server/server.js:100]:
```suggestion
```

File: app/bootstrap.js [hunk 3]
```diff
@@ -9,95 +9,138 @@
...
  */
 
+import _ from 'lodash';
 import React from 'react';
-import { render } from 'react-dom';
+import { render as reactDOMRender } from 'react-dom';
 import bows from 'bows';
 
-import './core/language'; // Set the language before loading components
+import i18n from './core/language'; // Set the language before loading components
```
Comment (yvon-dblg) [inline app/bootstrap.js:21]:
```suggestion
import i18n from './core/language';
```

**Comment by ydrisR** [comment]
Since we do not need documents per country, is it possible to simplify the code changes to determine the documents based on the language? Something as simple as a url based on the country code for example...

**Comment by MarineDBL** [comment]
We decided to keep the possibility to localize by country because it could be useful in the mid-term. Better to keep options open if that makes sense

**Review by ydrisR — COMMENTED** [review]
Looks good but there is a bug on the signup page: it is not possible to click on terms document link. When clicking the link it will check the "accept" checkbox :( 
Need to fix this and idealy to unit tests related to this new feture: basicaly verify the doc links are built based on language. 
We should also remember to add system tests specifics to this new feature

**Comment by yvon-dblg** [comment]
Reworked in https://github.com/mdblp/blip/pull/279

**Pull request closed without merging**